braindecode/models/util.py

```diff
@@ -3,11 +3,16 @@
 #
 # License: BSD (3-clause)
 
-import torch
 import numpy as np
+import torch
 from scipy.special import log_softmax
+from sklearn.utils import deprecated
 
 
+@deprecated(
+    "will be removed in version 1.0. Use EEGModuleMixin.to_dense_prediction_model method directly "
+    "on the model object."
+)
 def to_dense_prediction_model(model, axis=(2, 3)):
```

Comment:
Should we directly call `model.to_dense_prediction_model()` in this function ?

Reply:
I would not do that because it works for any `nn.Sequential` model -> for example if we remove logsoftmax with the current flow it won't work. We have to wait for the logsoftmax removal within EEGModuleMixin. After that I'm all in

Reply:
ok no problem 👍